src/test/java/com/nextstep/racingcargame/core/CarTest.java

```diff
@@ -25,27 +33,18 @@ void equalsAndHashCodeTest() {
         assertThat(group_one.equals(group_two)).isTrue();
     }
 
-    @Test
-    @DisplayName("차량 이동거리가 같은 Distance 객체 비교시 참을 리턴한다.")
-    void isSameTest() {
-        Car car = new Car(new Name(TEST_CAR_NAME), new Distance(CAR_START_POSITION_NUMBER));
-        assertThat(car.sameDistance(new Distance(CAR_START_POSITION_NUMBER))).isTrue();
-    }
-
     @Test
     @DisplayName("랜덤으로 발생한 숫자가 3 이하일경우 차량은 정지상태를 유지한다")
     void carStopTest() {
-        Car car = new Car(new Name(TEST_CAR_NAME), new Distance(CAR_START_POSITION_NUMBER));
-        car.move(new Stop());
-        assertThat(car).isEqualTo(new Car(new Name(TEST_CAR_NAME), new Distance(CAR_START_POSITION_NUMBER)));
+        basePositionedCar.move(new Stop());
+        assertThat(basePositionedCar).isEqualTo(new Car(new Name(TEST_CAR_NAME), new Distance(CAR_START_POSITION_NUMBER)));
     }
 
     @Test
     @DisplayName("랜덤으로 발생한 숫자가 4 이상일 경우 차량은 한칸 앞으로 움직인다.")
     void moveTest() {
-        Car car = new Car(new Name(TEST_CAR_NAME), new Distance(CAR_START_POSITION_NUMBER));
-        car.move(new ForceMove());
-        assertThat(car).isEqualTo(new Car(new Name(TEST_CAR_NAME), new Distance(MOVE_STEP)));
+        basePositionedCar.move(new ForceMove());
+        assertThat(basePositionedCar).isEqualTo(new Car(new Name(TEST_CAR_NAME), new Distance(MOVE_STEP)));
```

Comment:
basePositionedCar처럼 가독성을 위해서 비교대상이 되는 Car도 변수로 만들면 좋을 것 같아요 
```suggestion
        assertThat(basePositionedCar).isEqualTo(movedCar);
```